packages/api/src/routes/loginRoute.js

```diff
@@ -1,7 +1,8 @@
 const express = require('express');
 const router = express.Router();
 const loginController = require('../controllers/loginController');
-
+const checkGoogleJwt = require('../middleware/acl/checkGoogleJwt');
+router.post('/google', checkGoogleJwt, loginController.loginWithGoogle());
```

Comment:
Why are we checking for a jwt token for a login route?

Reply:
https://developers.google.com/identity/sign-in/web/sign-in